### PR TITLE
routing integration test fix.

### PR DESCRIPTION
**Pull request by bykoianko**
@dobriy-eeh @ygorshenin PTAL

### CODE REVIEW

File: routing/routing_integration_tests/routing_test_tools.cpp [hunk 1]
```diff
@@ -79,14 +79,20 @@ namespace integration
...
       return infoGetter.GetRegionCountryId(pt);
     };
 
+    shared_ptr<NumMwmIds> numMwmIds = make_shared<NumMwmIds>();
+    for (LocalCountryFile const & f : localFiles)
+      numMwmIds->RegisterFile(f.GetCountryFile());
+
     auto carRouter = make_unique<CarRouter>(index, countryFileGetter,
-                                            SingleMwmRouter::CreateCarRouter(countryFileGetter, nullptr, trafficCache, index));
+                                            SingleMwmRouter::CreateCarRouter(countryFileGetter, numMwmIds,
```
Comment (ygorshenin) [inline routing/routing_integration_tests/routing_test_tools.cpp]: Please, add a comment to the CreateCarRouter that numMwmIds should not be null, also, add a CHECK for that.

Comment (bykoianko) [inline routing/routing_integration_tests/routing_test_tools.cpp]: Done

**Comment by bykoianko** [comment]
@dobriy-eeh @ygorshenin PTAL

**Comment by dobriy-eeh** [comment]
LGTM

**Pull request merged**